### DISCUSSION TITLE
Dashboard / Homepage

### DISCUSSION THREAD

**Discussion by mistakia** [mistakia/nano-community#50] [Proposals]
**Representative Alerts**
- [x] Overweight
- [x] Offline
- [x] Behind
- [x] Low Uptime

**Live Network Stats**
- [x] Tx Throughput
- [x] Tx Backlog
- [x] Total Online Stake
- [x] Principal Reps
- [x] Total Reps
- [x] Peers
- [x] Reps to Confirm
- [x] Reps to Censor
- [x] Energy Usage TDP
- [ ] Effective bandwidth limit estimate
- [ ] Projected saturation throughput
- [ ] Settlement time / latency
- [ ] Receivable Online Weight (i.e. pending, unpocketed, etc)
- [ ] Principal Rep Online Weight
- [ ] Quorum Delta

**Ledger Stats**
- [ ] Periods
  - [ ] 1h
  - [ ] 24h
  - [ ] 3d
  - [ ] 7d
  - [ ] 1m
- [x] Confirmations
- [x] Send Volume
- [ ] Send Count
- [ ] Change Volume
- [ ] Change Count
- [ ] Top Accounts by Send Amount
- [ ] Top Accounts by Block Count
- [ ] Top Change Blocks by Balance / Weight
- [ ] Top Send Blocks by Amount

**Exchange Stats**
- [ ] Inflow Volume
- [ ] Outflow Volume
- [ ] Withdrawal count
- [ ] Deposit count
- [ ] Balance

**Development Events**
- [x] nano-node repo
- [ ] nano-docs repo
- [ ] nanodb-specification repo
- [ ] protocol repo

**Community Posts**
- [x] Nano Foundation Announcements
- [x] Sources
  - [x] Discord
  - [x] Forum
  - [x] Reddit

**Comment by kopeboy** [comment]
I would add # of Active accounts (that have transacted, sent or received), which is a key metric for tracking adoption

**Reply by mistakia** [reply]
currently have the # of active accounts per day here — https://nano.community/ledger

any suggestions on how you would want this metric communicated on the homepage?
